andela/travel_tool_front
# #62266939 Fix guesthouse flag

#### What does this PR do?
This PR ensures that guesthouse flags corresponds with its location.

#### Description of Task to be completed?
- Auto generate the country flag url based on guest house location

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- switch to this branch - `bg-fix-checklist-submissions-162008362`
- Install dependencies and start the  app - `make start`
-  [Start](https://github.com/andela/travel_tool_back/pull/174)  the backend.
- Navigate to [travela](http://travela-local.andela.com:3000) in your browser and login
- Go to the `DB` and on the `UserRoles` table, set yourself as a `Super Admin`
- Go to the browser, refresh the page,
- Navigate to the `manage residence` page - `Residence > Manage`
- The guesthouse flag should correspond to their location

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162266939](https://www.pivotaltracker.com/story/show/162266939)

#### Screenshots (if appropriate)
<img width="1440" alt="screenshot 2018-11-28 at 1 04 34 pm" src="https://user-images.githubusercontent.com/27569442/49150765-32130880-f30e-11e8-9f97-987442d1dec8.png">

#### Questions:
N/A
